src/index.js

```diff
@@ -1,11 +1,13 @@
 import "@babel/polyfill";
 import { bot } from "./bot";
+import { handleGetLastStake } from "./commands/get-last-stake";
 import { handleGetRichness } from "./commands/get-richness";
 import express from "express";
 import fetch from "node-fetch";
 import { APP_URL, PORT, BOT_USERNAME } from "./env";
 
 bot.command(["gr", `gr@${BOT_USERNAME}`], handleGetRichness);
+bot.command(["lr", `lr@${BOT_USERNAME}`], handleGetLastStake);
```

Comment:
Why is the command called `lr`? Following the unwritten convention I used with the `getRichness` command, `getLastStake` becomes `gls`.